### DISCUSSION TITLE
Error when trying to run bunx @snaplet/seed init

### DISCUSSION THREAD

**Discussion by Limzen** [prisma/prisma#25216] [Q&A] [answered]
![Screenshot 2024-09-12 211715](https://github.com/user-attachments/assets/24db9bdf-d1ed-4cec-aee2-f2e4a7b31b1c)

**Comment by ludralph** [comment, marked as answer]
Hi @Limzen 👋 

I am not able to reproduce this as the `bunx @snaplet/seed init` command runs correctly for me.
<img width="1385" alt="image" src="https://github.com/user-attachments/assets/71a96f74-ec2d-46de-9774-e1fb71e8f1f8">

Can you try clearing your bun cache?
```
bun pm cache rm
```
Then retry the command again. if you still run into issues, you can try swapping out bun fro npm and see if it works for you.
